addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4831,7 +4831,31 @@ msgctxt "#2104"
 msgid "Check the log for more information."
 msgstr ""
 
-#empty strings from id 2105 to 2202
+#. Heading text for yes/no dialog to ask for add-on install to get his font file
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2105"
+msgid "Missing font language support"
+msgstr ""
+
+#. Used to show notification about automatic language add-on installation, for here the used text
```

Comment:
Can you phrase this more in a way that describes the question being asked in the string? Also document what each {} means.